lib/ClassUtils.lua

```diff
@@ -33,7 +33,7 @@ function ClassUtils.makeConstructedClass(name, constructor)
 		ClassUtils.makeClass(
 		name,
 		function(data)
-			local instance = TableUtils.Clone(data)
+			local instance = TableUtils.clone(data)
```

Comment:
Why is there no corresponding metatable lookup for `ClassUtils` and `AsyncUtils`?

Reply:
they don't have any casing refactors. the lookup is just for adding the deprecated methods